.github/workflows/CI-python.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   ci-python:
     strategy:
       matrix:
-        packageDirectory: ["raiwidgets", "wrapped-flask"]
+        packageDirectory: ["raiwidgets", "wrapped-flask", "erroranalysis"]
```

Comment:
Do you also want to add this to the release workflow?

Reply:
yes